### ISSUE TITLE
Support ASTC HDR texture formats

### ISSUE THREAD

**Issue by devshgraphicsprogramming** [Devsh-Graphics-Programming/Nabla#505]
## Description
Appropriate feature needs to always be enabled if available.

Do not extend Limits of Features data structure, support will be know from query format usage props.

## Description of the related problem

Also CPU encode and decode routines must be added.

**Comment by ZzzhHe** [comment]
Hi, can I work on this?

**Comment by AnastaZIuk** [comment]
> Hi, I'd love to work on this issue! Is it still available?

Yes, join dc https://discord.gg/E6uYVdd3 and we will discuss details

**Comment by ZzzhHe** [comment]
Thanks! joined it. @alexzhhe is me.
